package.json

```diff
@@ -36,8 +36,8 @@
     "dev:www": "turbo --filter=www dev",
     "clean": "turbo clean && rm -rf node_modules",
     "lint": "turbo lint",
-    "format": "turbo format && prettier --write *.{cjs,md,json}",
-    "format:check": "turbo format:check && prettier --check *.{cjs,md,json}",
+    "format": "turbo format && prettier --write *.{cjs,md,json} --ignore-unknown --no-error-on-unmatched-pattern",
+    "format:check": "turbo format:check && prettier --check *.{cjs,md,json} --ignore-unknown --no-error-on-unmatched-pattern",
```

Comment:
Makes it not fail on the symlink:

![CleanShot 2023-02-19 at 09 50 25](https://user-images.githubusercontent.com/51714798/219938373-4b9181e2-dff7-4414-8ace-3304513569be.png)